lib/economic/endpoint.rb

```diff
@@ -8,6 +8,10 @@ class Economic::Endpoint
   def_delegator "client.globals", :log_level, :log_level=
   def_delegator "client.globals", :log, :log=
 
+  def initialize(app_identifier = nil)
```

Comment:
Can we add some documentation about what `app_identifier` is meant to be? Has E-conomic written something somewhere we can copy/paste and/or link to?